prow/cmd/branchprotector/README.md

```diff
@@ -0,0 +1,7 @@
+Branchprotector enforces the github branch proection declared in a config file.
+
+Typical pattern would be to deploy
```

Comment:
Deploy what and how? Also the 'Usage' section seems a little incomplete.

Reply:
uh, how did this get into this PR??